crates/context/config/src/client.rs

```diff
@@ -71,27 +67,24 @@ pub trait Transport {
     ) -> Result<Vec<u8>, Self::Error>;
 }
 
-impl<L: Transport, R: Transport, T: Transport> Transport for TransportChoice<L, R, T> {
-    type Error = TransportError<L::Error, R::Error, T::Error>;
+impl<L: Transport, R: Transport, T: Transport> Transport for Either<L, Either<R, T>> {
```

Comment:
you should only need to implement this for `Either<L, R>`

at which point, `Either<L, Either<R, O>>` will also be satisfied because

`Either<R, O>` also impls `Transport`

just change it back to the way it was before on master, it should work

Reply:
Reverted